lib/beaker/host/mac/pkg.rb

```diff
@@ -25,7 +27,35 @@ def generic_install_dmg(dmg_file, pkg_base, pkg_name)
     end
     dmg_name = File.basename(dmg_file, '.dmg')
     execute("hdiutil attach #{dmg_name}.dmg")
-    execute("installer -pkg /Volumes/#{pkg_base}/#{pkg_name} -target /")
+
+    # First check if the specific package exists, otherwise use wildcard
+    specific_pkg_path = "/Volumes/#{pkg_base}/#{pkg_name}"
+    execute("test -f #{specific_pkg_path}", :accept_all_exit_codes => true) do |result|
+      if result.exit_code == 0
+        # $pkg_name package found so install it
+        execute("installer -pkg #{specific_pkg_path} -target /")
+      else
+        # else find and install the first *.pkg file in the volume
+        execute(<<~SCRIPT
```

Comment:
Perhaps leave out the brace since it's not needed?
```suggestion
        execute <<~SCRIPT
```

Reply:
Removed un-needed paranthesis.